test/services/EtmpRegimeServiceTest.scala

```diff
@@ -35,63 +53,33 @@ class EtmpRegimeServiceTest extends BaseSpec {
   implicit val hc: HeaderCarrier = HeaderCarrier(sessionId = Some(SessionId(s"session-${UUID.randomUUID}")))
 
   val mockEtmpConnector: EtmpConnector = mock[EtmpConnector]
-  val safeId: String = "XV1234567890123"
+  val mockAuthConnector: AuthConnector = mock[AuthConnector]
+  val mockEnrolmentStore: EnrolmentStoreConnector = mock[EnrolmentStoreConnector]
+  val safeId: String = "XE0001234567890"
   val regime: String = "AWRS"
 
-  object TestEtmpRegimeService extends EtmpRegimeService(mockEtmpConnector)
+  object TestEtmpRegimeService extends EtmpRegimeService(mockEtmpConnector, mockEnrolmentStore, mockAuthConnector )
 
 
   "awrsRegime" should {
-    "successfully return an empty reference" in {
-      when(mockEtmpConnector.awrsRegime(Matchers.any(), Matchers.any())(Matchers.any()))
-        .thenReturn(Future.successful(HttpResponse(OK, None)))
-      val result = TestEtmpRegimeService.getRegimeRefNumber(safeId, regime)
-
-      await(result) shouldBe None
-    }
-
     "successfully return a regimeRefNumber" in {
-      when(mockEtmpConnector.awrsRegime(Matchers.any(), Matchers.any())(Matchers.any()))
-        .thenReturn(Future.successful(HttpResponse(OK, Some(Json.parse(
-          """
-            |{"regimeIdentifiers": [
-            |{
-            |  "regimeName": "String",
-            |  "regimeRefNumber": "AWRS"
-            |}
-            |]}
-          """.stripMargin)))))
-      val result = TestEtmpRegimeService.getRegimeRefNumber(safeId, regime)
+      when(mockEtmpConnector.awrsRegime(Matchers.any())(Matchers.any()))
```

Comment:
RED - I know this test was refactored but it's name doesn't correspond to what it's testing. The function awrsRegime is mocked out here so it's not what's being tested.